tests/test_data/expected-semver-ranges-comma-separated.txt

```diff
@@ -0,0 +1,1441 @@
+>=8.0, <8.4.5
```

Comment:
https://www.python.org/dev/peps/pep-0440/#version-specifiers
`Whitespace between a conditional operator and the following version identifier is optional, as is the whitespace around the commas.` it's optional on python but what do snyk think about it?

Reply:
```suggestion
>= 8.0, < 8.4.5
```

Reply:
This is something to consider if we opt to adding the ability to do `to_semver` for different language dialects. Right now, out of scope.